app.py

```
@@ -45,6 +45,16 @@ class QATCH:
     ###########################################################################
     def __init__(self, argv=sys.argv):
+        if getattr(sys, "frozen", False):
+            userpath = os.path.expandvars("%USERPROFILE%")
+            docspath = os.path.join(userpath, "Documents", "QATCH nanovisQ")
+            if os.path.isdir(docspath):
+                current_cwd = os.getcwd()
+                if current_cwd != docspath:
+                    try:
+                        os.chdir(docspath)
+                    except Exception as e:
+                        raise e
```

Ruff (0.15.0)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)